.github/workflows/pull-requests.yml

```diff
@@ -13,6 +13,24 @@ name: PR Workflow
 
 on:
   pull_request:
+    paths:
+      - '.github/**'
+      - 'env.example'
+      - '.husky/**'
+      - 'scripts/**'
+      - 'package.json'
+      - 'tsconfig.json'
```

Comment:
Remove all these paths. This means all the Github actions in this file will only be run when these files are modified. Please research the usage. We stated that we wanted the files defined in only one location